README.md

```diff
@@ -63,22 +63,22 @@ const model = {
   // true if the request resolves or rejects
   isCompleted: false,
 
-  // true if the request is cancled
+  // true if the request is canceled
   isCanceled: false,
 
   // true if the request rejected, and is being called again
   isRetrying: false,
 
-  // start time of request in milliseconds UTC
+  // start time of request in milliseconds
```

Comment:
Since I've never really used these properties in practice I'm not sure if it matters, but _should_ we be basing our timestamps off UTC?

On one hand, this timestamp information is really only valuable in the application where the waiter is operating and in that respect local time seems fine. However, perhaps it would be more robust if one needed to back out a date if these timestamps were UTC?

This seems like a small consideration, I just thought it was interesting to consider the options here.

Reply:
It is using the getTime() from Date, so it is UTC. I added it back to the docs.

Reply:
I'm getting myself turned around between default `Date` and moment. 🤦‍♂
Moment assumes local time unless specifying UTC.